app/controllers/chores_controller.rb

```diff
@@ -17,6 +19,15 @@ def create
     redirect_to @chore
   end
 
+  def update
+    @chore = Chore.find(params[:id])
+    @chore.lastdone = Date.today
+    @chore.nextdue = @chore.lastdone + @chore.interval_days
+    @chore.updated_at = Time.now
```

Comment:
this should automatically get updated when this action is called. is that not the case?